### DISCUSSION TITLE
Split panes with alt+shift+(-|+)

### DISCUSSION THREAD

**Discussion by black-snow** [wezterm/wezterm#3922] [Q&A]
I've just installed wezterm as I wanted a nice emulator with splitting functionality without any extra fuzz. But it doesn't seem to have any (GUI) configuration settings? Do I have to look up default key bindings on the website?

My real question is about how to split vertically/horizontally with alt+shift+"+"/"-". I tried to go with https://wezfurlong.org/wezterm/config/lua/keyassignment/SplitVertical.html and replace the `key` with `+` and `-` respectively as well as dropping `CTRL` from the mods. But it won't work.  
I also tried to leave the `mods` as-is but to no avail. Nothin happens (acutally alt+shift+- seems to take the last thing from the history and remove the first thing split by space ...).

**Comment by nekowinston** [comment]
This config works for me:

```lua
local wezterm = require("wezterm")

local c = {}
if wezterm.config_builder then
  c = wezterm.config_builder()
  c:set_strict_mode(true)
end

local act = wezterm.action
c.keys = {
  {
    key = "-",
    mods = "SHIFT|ALT",
    action = act.SplitVertical({ domain = "CurrentPaneDomain" }),
  },
  {
    key = "=",
    mods = "SHIFT|ALT",
    action = act.SplitHorizontal({ domain = "CurrentPaneDomain" }),
  },
}

return c
```

If this won't work, maybe something in your OS is intercepting Alt+Shift?

**Reply by black-snow** [reply]
Thanks for the quick reply. Sadly, it doesn't work.  
I'm on Ubuntu lunar, pretty much vanilla. I don't see any global shortcuts in GNOME that might interfere.

**Reply by nekowinston** [reply]
Where's your config located? Can you share it here or create a gist?

**Reply by black-snow** [reply]
Here's my `~/.wezterm.lua`:

```lua
-- Pull in the wezterm API
local wezterm = require 'wezterm'

-- This table will hold the configuration.
local config = {}

-- In newer versions of wezterm, use the config_builder which will
-- help provide clearer error messages
if wezterm.config_builder then
  config = wezterm.config_builder()
  config:set_strict_mode(true)
end

-- This is where you actually apply your config choices

config.keys = {
  -- This will create a new split and run your default program inside it
  {
    key = '-',
    mods = 'SHIFT|ALT',
    action = wezterm.action.SplitHorizontal { domain = 'CurrentPaneDomain' },
  },
}
config.keys = {
  -- This will create a new split and run your default program inside it
  {
    key = '+',
    mods = 'SHIFT|ALT',
    action = wezterm.action.SplitVertical { domain = 'CurrentPaneDomain' },
  },
}
config.color_scheme = 'Solarized (dark) (terminal.sexy)'

return config
```

**Reply by jasper-war** [reply]
I had to use key = 'mapped:+' in order for it to function.
